framework/src/play/i18n/MessagesPlugin.java

```diff
@@ -25,27 +24,13 @@ public class MessagesPlugin extends PlayPlugin {
     public void onApplicationStart() {
         includeMessageFilenames.clear();
         Messages.defaults = new Properties();
-        for (VirtualFile module : Play.modules.values()) {
-            VirtualFile messages = module.child("conf/messages");
-            if (messages != null && messages.exists()
-                    && !messages.isDirectory()) {
-                Messages.defaults.putAll(read(messages));
-            }
-        }
-        VirtualFile appDM = Play.getVirtualFile("conf/messages");
+        File appDM = Play.getVirtualFile("conf/messages");
```

Comment:
I'd not abbreviate `appDM` (`defaultMessagesFile`), same for `appM` below. The symbol is short lived, so it does not matter much. Nitpick.

Reply:
renamed `appDM` / `appM`.